src/css.js

```diff
@@ -113,6 +113,20 @@ function getWidthOrHeight( elem, name, extra ) {
 		styles = getStyles( elem ),
 		isBorderBox = jQuery.css( elem, "boxSizing", false, styles ) === "border-box";
 
+	// Support: IE11+
+	// Fix for edge case in IE 11. See https://github.com/jquery/jquery/issues/1764
+	if ( !support.reliableFullscreenBox && window.top !== window.self &&
+		document.msFullscreenElement ) {
+		// Support: IE<=11+
+		// Running getBoundingClientRect on a
+		// disconnected node in IE throws an error
+		if ( !elem.getClientRects().length ) {
+			val *= 100;
```

Comment:
You skip this assignment and just do anything in the case where `.length` is truthy. Disconnected elements have `offsetWidth === 0` (see [the spec](http://dev.w3.org/csswg/cssom-view/#dom-htmlelement-offsetwidth)) so `val === 0` at this point for such elements; multiplying it by 100 won't change anything.